benches/benches.rs

```diff
@@ -110,3 +110,27 @@ fn bench_iter_ones_all_ones(b: &mut Bencher) {
     });
 }
 
+#[bench]
+fn bench_insert_range(b: &mut Bencher) {
+    const N: usize = 1_000_000;
+    let mut fb = FixedBitSet::with_capacity(N);
+
+    b.iter(|| {
+        test::black_box({
+            fb.insert_range(..);
+        });
+    });
```

Comment:
Oh I see that the existing benchmarks already use black_box in weird ways. Rest assured that should all be cleaned up. `b.iter()` already blackboxes the return value of the closure.